partd/pandas.py

```diff
@@ -14,6 +15,10 @@
 
 dumps = partial(pickle.dumps, protocol=pickle.HIGHEST_PROTOCOL)
 
+_NO_EXTENSION = 0
```

Comment:
Agreed on not using an emum (less technology seems better).  Maybe just use strings instead?  They will probably be easier to debug and I don't think there will be a significant performance penalty.